### PR TITLE
structurizr-cli 1.8.0

### PR DESCRIPTION
**Pull request by gilbertotcc**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Update `structurizr-cli` to version 1.8.0.

### CODE REVIEW

File: Formula/structurizr-cli.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class StructurizrCli < Formula
   desc "Command-line utility for Structurizr"
   homepage "https://structurizr.com"
-  url "https://github.com/structurizr/cli/releases/download/v1.7.0/structurizr-cli-1.7.0.zip"
-  sha256 "3f5b8b35d85f0d9dbddb874c644af3ed08f46bcde4c6d8161116e72c60360171"
+  url "https://github.com/structurizr/cli/releases/download/v1.7.0/structurizr-cli-1.8.0.zip"
```
Comment (issyl0) [inline Formula/structurizr-cli.rb:4]:
```suggestion
  url "https://github.com/structurizr/cli/releases/download/v1.8.0/structurizr-cli-1.8.0.zip"
```

**Review by issyl0 — CHANGES_REQUESTED** [review]
This is almost there! Just a tiny typo in the `url`.

**Comment by gilbertotcc** [comment]
> This is almost there! Just a tiny typo in the `url`.

@issyl0 *Ops!* Sorry for the typo and thx for the suggestion.

**Review by issyl0 — APPROVED** [review]
Thanks for your contribution to Homebrew, @gilbertotcc!

**Pull request merged**